### DISCUSSION TITLE
No signal, not even noise. Antenna not suitable?

### DISCUSSION THREAD

**Discussion by larsen0815** [AlexandreRouma/SDRPlusPlus#1435] [General]
Hi,

I have a generic RTL-SDR that I'm already using with a Raspberry Pi to collect ADS-B data. Now I wanted to try if I could also receive ATC audio, so I plugged the dongle into my Windows 11 laptop, and started SDR++ v1.2.0. The dongle is listed in the source dropdown, so seems to be supported. The antenna has a length of ~12 cm, so this might be a problem I guess.

For testing purposes, frequency is set to a local FM radio station. See screenshot for settings.
When I click on the Play button, nothing happens.

Is this to be expected or should I at least get some noise?

Lars

![Clipboard-1](https://github.com/AlexandreRouma/SDRPlusPlus/assets/2345512/125a7ece-376d-4373-8552-e22faaf05b90)

**Comment by KubaPro010** [comment]
Click `Refresh` and maybe then it will work (stop it before clicking refresh)

**Comment by AlexandreRouma** [comment]
you didn't install the drivers with zadig.

**Reply by KubaPro010** [reply]
either that or just that what i said, i also sometimes get that

**Reply by AlexandreRouma** [reply]
no this is clearly just no zadig

**Comment by larsen0815** [comment]
There is no information in the readme that this is necessary =/

I have followed the instructions at https://www.rtl-sdr.com/getting-the-rtl-sdr-to-work-on-windows-10/ using Zadig from https://zadig.akeo.ie and now it's working fine, **thanks!** =)

Probably should be added to the readme and if possible be recognized by the program itself. I wouldn't have guessed missing drivers could be a problem as the device was shown in the GUI. Difference now is the entry having a number instead of "No Serial".

**Reply by AlexandreRouma** [reply]
If the readme had to tell you how to install the drivers for ever SDR device that SDR++ supports it would be the length of the bible.

**Comment by larsen0815** [comment]
But it could tell you *that* the drivers have to be installed.
